sentry/src/db/analytics.rs

```diff
@@ -0,0 +1,95 @@
+use crate::db::DbPool;
+use crate::Session;
+use bb8::RunError;
+use chrono::Utc;
+use primitives::analytics::{AnalyticsQuery, AnalyticsResponse};
+
+pub async fn get_analytics(
+    query: AnalyticsQuery,
+    channels: Option<String>,
```

Comment:
Can't we just deserialize this? It's very hard to reason about this `String` value. Also apparently it's a single `channel`

Reply:
There is no point to deserialize, it either an appropriate value that returns some data or nothing

Reply:
Ok, that sounds right. Can we at least rename it to `channel` as it's only querying a single channel